src/ontology/mondo.Makefile

```diff
@@ -268,8 +269,42 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
-	
+#############################################
+##### One-time scripts ######################
+#############################################
+# MedGen conflicts (Aug 2023) pipeline
+# - https://github.com/monarch-initiative/mondo-ingest/issues/273
+.PHONY: address-medgen-conflicts-aug2023
+address-medgen-conflicts-aug202x3: address-medgen-conflicts-aug2023-deletes address-medgen-conflicts-aug2023-adds
+
+# - MedGen conflicts: dependencies
+tmp/July2023_CUIReports_FromMedGentoMondo.xlsx:
+	mkdir -p tmp && wget "https://github.com/monarch-initiative/mondo-ingest/files/12029712/July2023_CUIReports_FromMedGentoMondo.xlsx" -O $@
+
+# - MedGen conflicts: analysis
+tmp/mondo-edit.obo.tmp.diff: mondo-edit.obo.tmp
+	-diff mondo-edit.obo mondo-edit.obo.tmp > $@
+
+tmp/report-qc-medgen-conflicts-update-diff.tsv: tmp/mondo-edit.obo.tmp.diff
+	python ../scripts/medgen_conflicts_removals_diff_analysis.py -i tmp/mondo-edit.obo.tmp.diff -o $@
+
+# - MedGen conflicts: deletes
+.PHONY: address-medgen-conflicts-aug2023-deletes
+address-medgen-conflicts-aug2023-deletes: mondo-edit.obo.tmp
+	mv mondo-edit.obo.tmp mondo-edit.obo
+
+mondo-edit.obo.tmp: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx mondo-edit.obo
+	python ../scripts/medgen_conflicts_removals.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -I mondo-edit.obo -o $@
+
+# - MedGen conflicts: adds
+.PHONY: address-medgen-conflicts-aug2023-adds
+address-medgen-conflicts-aug2023-adds: $(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv $(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv
+
+$(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx
+	python ../scripts/medgen_conflicts_add_xrefs.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -o $@
 
+$(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv:
```

Comment:
## Next & final step: xref additions from MedGen mappings
As I wrote in [the OP](https://github.com/monarch-initiative/mondo/pull/6560#issue-1850743276), my next step is: 
> 3. Add new xrefs generated from [MedGen ingest](https://github.com/monarch-initiative/medgen): which will mainly come from the FTP file MedGenIDMappings.txt, but will probably require that whole ingest to be working to generate.

This depends on the medgen ingest working properly, and it's in pretty good shape I think, so I don't anticipate much longer until this is done too.